app/components/forms/session-speaker-form.js

```diff
@@ -424,6 +424,11 @@ export default Component.extend(FormMixin, {
               type   : 'regExp',
               value  : validTwitterProfileUrlPattern,
               prompt : this.l10n.t('Please enter a valid twitter profile url')
+            },
+            {
+              type   : 'regExp',
+              value  : protocolLessValidUrlPattern,
+              prompt : this.l10n.t('Please enter a valid url')
```

Comment:
I am quite not sure if we can put two same kind of rules for validation. `regExp` in this case. I am AFK as of now and hence can't test it.

Reply:
@Anupam-dagar I tested it for both cases, It throws an appropriate error when needed